pixi.toml

```diff
@@ -33,14 +33,15 @@ hdf4 = "*"
 isodate = "*"
 itsdangerous = "*"
 keyring = "*"
+libgfortran5 = ">=14"
 libtiff = "*"
 lxml = "*"
 markdown = "*"
 matplotlib = ">=3.5.3"
 metpy = "*"
 multidict = "*"
 netcdf4 = "*"
-numpy = ">=2"
+numpy = ">=2.2.4,<3"
```

Comment:
Numpy >=3 does not exist yet. Why do you presume it to be incompatible?

Reply:
we can also wait until 3 exists, but yeah lower limit in our case would work